src/gantt.ts

```diff
@@ -895,7 +958,21 @@ export class Gantt implements IVisual {
                 extraInformation,
                 highlight,
                 task
-            } = this.createTask(values, index, hasHighlights, categoricalValues, color, completion, categoryValue, endDate, duration, taskType, selectionBuilder, wasDowngradeDurationUnit, stepDurationTransformation);
+            } = this.createTask({
```

Comment:
When you call a method, don't pass a giant object "in place", create a separate variable and pass it here. Please follow the same pattern in other places.